welcome/overview/is-it-for-you.md

```diff
@@ -1,37 +1,34 @@
-[title]: - "Is High Throughput Computing for You?"
+[title]: - "Is the Open Science Grid for You?"
 
 
-## Ideal Charateristics for Distributed High Throughput Computing (HTC)
+The [Open Science Grid][osg] is a national network that offers a strategic advantage 
```

Comment:
Maybe something like "The Open Science Grid is an nationally-funded consortium of computing resources at more than one hundred institutional partners that, together, offer a strategic advantage for computing work that ..."
We need to avoid "national", but can imply it.